cdap-ui/app/cdap/services/AlertStatus.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+enum ALERT_STATUS {
+  Success = 'success',
```

Comment:
Since this is purely used in Alert component can we co-locate this with Alert component?
Would be better if we convert the Alert to typescript and add the enum to the same class as a static variable. That way components using Alert don't need to import this as an extra they can just use it as a static variable.

Reply:
As we discussed, I've created a JIRA and TO DO comment to track converting the component to TS in a separate PR.